Android SDK version for automatic distributed tracing (#8506)
getsentry/sentry-docs: src/platform-includes/distributed-tracing/how-to-use/android.mdx

> If you use the current version of our Android SDK, distributed tracing works out of the box for any HTTP request sent with OkHttp, Apollo or Apollo 3, if our integration for it is enabled.

Reviewer: Is this version # correct here? Its better to use a specific version # when possible, since it's always changing!

```suggestion
For Sentry Android SDK versions `6.25.x` and higher, distributed tracing works out of the box for any HTTP request sent with OkHttp, Apollo, or Apollo 3, if the integration for it is enabled.
```
